docs/sources/administration/plugin-management/index.md

```diff
@@ -61,13 +65,11 @@ To prevent users from seeing an app plugin, refer to [this permissions scenarios
 
 ## Plugin catalog
 
-The Plugin catalog allows you to browse and manage plugins from within Grafana. Only Grafana server administrators and organization administrators can access and use the plugin catalog. The following access rules apply depending on the user role:
+The Plugin catalog allows you to browse and manage plugins from within Grafana. Only Grafana server administrators and organization administrators can access and use the plugin catalog. For more information about Grafana Roles and Permissions, refer to [Roles and permissions]({{< relref "../administration/roles-and-permissions" >}}). The following access rules apply depending on the user role:
 
-| Org Admin | Server Admin | Permissions                                                                                 |
-| --------- | ------------ | ------------------------------------------------------------------------------------------- |
-| &check;   | &check;      | <ul><li>Can configure app plugins</li><li>Can install/uninstall/update plugins</li></ul>    |
-| &check;   | &times;      | <ul><li>Can configure app plugins</li><li>Cannot install/uninstall/update plugins</li></ul> |
-| &times;   | &check;      | <ul><li>Cannot configure app plugins</li><li>Can install/uninstall/update plugins</li></ul> |
+- If you are an **Org Admin**, you can configure app plugins but you cannot install/uninstall/update them.
+- If you are a **Server Admin**, you cannot configure app plugins but you can install/uninstall/update them.
+- If you are both (**Org + Server Admin**), you can configure app plugins and also install/uninstall/update them.
```

Comment:
I still feel like this is confusing, but this time because it implies that it is only about app plugins, whereas install/uninstall/update refers to **all** plugins. 

@josmperez perhaps you could suggest something?

I had originally suggested:
> - The **server admin** role is required to install/uninstall/update plugins.
> - The **org admin** role is required to configure app plugins.

Maybe a change to the preceding paragraph would also help